desc/compute/_core.py

```diff
@@ -2791,6 +2973,43 @@ def _phi_t(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="phi_rzz",
+    label="\\partial_{\\rho \\zeta \\zeta} \\phi",
+    units="rad",
+    units_long="radians",
+    description="Toroidal angle in lab frame, second derivative wrt radial and "
+    "toroidal coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["omega_rzz"],
+)
+def _phi_rzz(params, transforms, profiles, data, **kwargs):
+    data["phi_rzz"] = data["omega_rzz"]
```

Comment:
Should instead add `alias="phi_rzz"` to the "omega_rzz" compute functions

Reply:
I actually prefer keeping these distinct, see https://github.com/PlasmaControl/DESC/pull/1175#discussion_r1713107215

Reply:
All this crap has since been removed!